lca-cli/postpivot/postpivot.go

```diff
@@ -175,6 +175,18 @@ func (p *PostPivot) recert(ctx context.Context, seedReconfiguration *clusterconf
 		return err
 	}
 
+	ctxWithTimeout, cancel := context.WithTimeout(ctx, 10*time.Minute)
+	_ = wait.PollUntilContextCancel(ctxWithTimeout, time.Second, true, func(ctx context.Context) (bool, error) {
+		p.log.Info("pulling recert image")
+		if _, err := p.ops.RunInHostNamespace("podman", "pull", seedClusterInfo.RecertImagePullSpec); err != nil {
+			p.log.Warnf("failed to pull recert image, will retry, err: %s", err.Error())
+			return false, nil
+		}
+		// TODO: add waiting for block device with label
+		return true, nil
+	})
+	defer cancel()
```

Comment:
wondering if `cancel()` should be called before the `wait`? 

...just to ensure that the cancellation function is invoked even if an error occurs before the polling loop starts

Reply:
defer should be added before it for sure, will change